src/components/EntityInfo/EntityInfo.tsx

```diff
@@ -1,7 +1,7 @@
 import "./_entity-info.scss";
 
 type Props = {
-  data: { [key: string]: string | JSX.Element };
+  data: { [key: string]: string | number | JSX.Element };
```

Comment:
It looks like you could probably just use `data: { [key: string]: ReactNode };`.